test_data/test_data.rb

```diff
@@ -1,11 +1,10 @@
 module TestData
-  EXPECTED_APPROXIMATE = 'Approximate round trip times in milli-seconds:'.freeze
-  EXPECTED_AVERAGE = 'Average = 0ms'
-  EXPECTED_DESKTOP = 'Pinging DESKTOP-MRO0O4C'
-  EXPECTED_STATISTICS = 'Ping statistics for ::1:'
-  EXPECTED_HELLO = 'Hello World'
   TEMP_FILE_PATH = "C:\\test_folder\\temp_file.txt"
-  PING_LOCALHOST = 'ping localhost'
-  REGTEMPLATE = /Minimum = \d+ms, Maximum = \d+ms, Average = \d+ms/
-  OUTWRITEPLUS = 'w+'
+  EXPECTED_DATA = {
+    :EXPECTED_APPROXIMATE => 'Approximate round trip times in milli-seconds:',
+    :EXPECTED_AVERAGE => 'Average = 0ms',
+    :EXPECTED_DESKTOP => 'Pinging DESKTOP-MRO0O4C',
+    :EXPECTED_STATISTICS => 'Ping statistics for ::1:',
+    :EXPECTED_HELLO => 'Hello World',
```

Comment:
in hash if you use symbols - don't use all caps
for instance
HASH = {
    :key => 'value'
}

Reply:
Done.